lib/url.js

```diff
@@ -120,6 +120,16 @@ export function parseEmbedUrl (href) {
         }
       }
     }
+
+    if (hostname.includes('peertube.tv') || hostname.includes('bitcointv.com')) {
```

Comment:
`hostname.includes` is still prone to matching malicious domains since any domain could contain a subdomain that matches this.

You needs to use `endsWith` or an exact match.